packages/safe-ds-lang/src/language/generation/safe-ds-python-generator.ts

```diff
@@ -1083,6 +1090,16 @@ export class SafeDsPythonGenerator {
         throw new Error('Callable of provided call does not exist or is not a declaration.');
     }
 
+    private getClassQualifiedNameForMember(expression: SdsClassMember): string {
+        const classMemberPath = [this.getPythonNameOrDefault(expression)];
+        let enclosingClass = AstUtils.getContainerOfType(expression.$container, isSdsClass);
+        while (enclosingClass) {
+            classMemberPath.push(this.getPythonNameOrDefault(enclosingClass));
```

Comment:
Could also `unshift` here to insert at the start. Then you don't need to reverse at the end.